bundles/action/org.openhab.action.tinkerforge/META-INF/MANIFEST.MF

```diff
@@ -11,10 +11,12 @@ Bundle-ManifestVersion: 2
 Bundle-Description: This is the TinkerForge action of the open Home Aut
  omation Bus (openHAB)
 Import-Package: org.apache.commons.lang,
- org.openhab.core.scriptengine.action,
+ org.openhab.binding.tinkerforge.ecosystem,
+ org.openhab.binding.tinkerforge.internal.model,
```

Comment:
Absolutely right and we should change this (out of the scope of this Bugfix). What i changed was simply removing the require-bundle deep and add the import-package instead. @theoweiss could you please take care of this and create another PR moving the externally required classes in the appropriate package?